src/Controls/src/Core/BindableProperty.cs

```diff
@@ -41,8 +41,6 @@ public sealed class BindableProperty
 		static readonly Dictionary<Type, TypeConverter> KnownTypeConverters = new Dictionary<Type, TypeConverter>
 		{
 			{ typeof(Uri), new UriTypeConverter() },
-			{ typeof(Color), new ColorTypeConverter() },
```

Comment:
Do we want to remove this one too ? or just the easing ?

Reply:
we want to remove this one